src/RunTests/README.md

```diff
@@ -0,0 +1,82 @@
+# Microsoft.Build.RunVSTest
+
+The `Microsoft.Build.RunVSTest` MSBuild SDK adds support for running tests from MSBuild, similarly to how one would use `dotnet test`.
+
+## For projects that cannot use package references such as vcxproj. Usage in `Directory.Packages.Props` 
+In your global.json add the following:
+```json
+{
+  "msbuild-sdks": {
+	"Microsoft.Build.RunVSTest": "1.0.0"
+  }
+}
+```
+In your ..vcxproj file
+```xml
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Sdk Name="Microsoft.Build.RunVSTest"/>
+  ...
+</Project>
+```
+
+## For projects that use packages references. In your `Directory.Packages.props`:
+```xml
+<Project>
```

Comment:
are we missing here the `<UseMSBuildTestInfrastructure>true</UseMSBuildTestInfrastructure>` ?

Reply:
So the non tests project will import the empty `Test` target and won't fail for target not found for run at solution level?

Reply:
> <UseMSBuildTestInfrastructure>true</UseMSBuildTestInfrastructure>

updated readme to include